xml/gnome_custom.xml

```diff
@@ -133,17 +121,15 @@
    <itemizedlist>
     <listitem>
      <para>
-      <guimenu>Language</guimenu>. Choose a language from the list and confirm
-      with <guimenu>Done</guimenu>. To make the change take effect, you need to
-      restart the desktop afterwards.
+      <guimenu>Language</guimenu>. Choose a language from the list. To make the
+      change take effect, you need to restart the desktop afterwards.
      </para>
     </listitem>
     <listitem>
      <para>
       <guimenu>Formats</guimenu>: To change the formats for date, number,
-      currency and related options, choose a country from the list and confirm
-      with <guimenu>Done</guimenu>. To make the change take effect, you need to
-      restart the desktop afterwards.
+      currency and related options, choose a country from the list. To make the
+      change take effect, you need to restart the desktop afterwards.
```

Comment:
Can we clarify "restart the desktop"? Does it mean log out and log back in, or is a reboot necessary?

Reply:
I just checked: Obviously one just needs to restart the desktop session and GNOME prompts you to do so. If you accept, you will be logged out the desktop and have to log back in.